pymongo/monitor.py

```diff
@@ -319,7 +320,7 @@ def _get_seedlist(self):
         try:
             resolver = _SrvResolver(self._fqdn,
                                     self._settings.pool_options.connect_timeout,
-                                    self._srv_service_name)
+                                    self._settings._srv_max_hosts)
```

Comment:
This looks off. Don't we need to pass *both* `srv_service_name` and `srv_max_hosts`?

Reply:
Oh oops, yeah that is a typo. Fixed.